cpp/ql/lib/semmle/code/cpp/Function.qll

```diff
@@ -338,10 +339,6 @@ class Function extends Declaration, ControlFlowNode, AccessHolder, @function {
     exists(FunctionCall call |
       call.getEnclosingFunction() = this and call.getTarget() = f and call = l
     )
-    or
```

Comment:
Is this useless as `DestructorCall` is a subtype of `FunctionCall`?

Reply:
Exactly: https://github.com/github/codeql/blob/main/cpp/ql/lib/semmle/code/cpp/exprs/Call.qll#L461